generator/sbpg/targets/test_rust.py

```diff
@@ -0,0 +1,92 @@
+#!/usr/bin/env python
+# Copyright (C) 2015 Swift Navigation Inc.
+# Contact: Joshua Gross <josh@swiftnav.com>
+#
+# This source is subject to the license found in the file 'LICENSE' which must
+# be be distributed together with this source. All other rights reserved.
+#
+# THIS CODE AND INFORMATION IS PROVIDED "AS IS" WITHOUT WARRANTY OF ANY KIND,
+# EITHER EXPRESSED OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE IMPLIED
+# WARRANTIES OF MERCHANTABILITY AND/OR FITNESS FOR A PARTICULAR PURPOSE.
+
+"""
+Generator for c tests target.
+"""
+
+from sbpg.targets.templating import *
+from sbpg.targets.rust import *
+import base64
+
+TEST_TEMPLATE_NAME = "sbp_tests_template.rs"
+#CHECK_SUITES_TEMPLATE_NAME = "sbp_c_suites.h.j2"
+#CHECK_MAIN_TEMPLATE_NAME = "sbp_c_main.c.j2"
+
+def b64_decode(field):
+    print("Decoding '{}'".format(field))
+    b = base64.standard_b64decode(field)
+    return [ str(ord(ch)) for ch in b ]
+
+def stringType(value):
+    return type(value) == str
+
+def arrayType(value):
+    return type(value) == list
+
+def dictType(value):
+    return type(value) == dict
+
+def floatType(value):
+    return type(value) == float
+
+def isEmpty(value):
+    return len(value) == 0
+
+def strEscape(value):
+    return "\"{}\".to_string()".format(value)
+
+def toStr(value):
+    return str(value)
+
+def modName(value):
+    return value.split('.')[1]
+
+JENV.filters['b64_decode'] = b64_decode
```

Comment:
Let's try to normalize the naming if these template functions

Reply:
Done here and in the C test generator.